bin/ivar_variants_to_vcf.py

```diff
@@ -11,7 +11,10 @@ def parse_args(args=None):
 
     parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
     parser.add_argument('FILE_IN', help="Input tsv file.")
-    parser.add_argument('FILE_OUT', help="Full path to output vcf file")
+    parser.add_argument('FILE_OUT', help="Full path to output vcf file.")
+    parser.add_argument('-po', '--pass_only', dest="PASS_ONLY", help="Only output variants that are tagged as PASS.",action='store_true')
```

Comment:
Only output variants that are tagged as PASS &rarr; Only output variants that pass all filters  ?

Reply:
Fixed https://github.com/nf-core/viralrecon/pull/93/commits/a031c5dffe45265a3c6f905712d84736dd9cebfb